.github/pr-custom-review.yml

```diff
@@ -8,12 +8,24 @@ rules:
     check_type: changed_files
     condition:
       include: .*
-      # excluding files from 'CI team' rules
-      exclude: ^\.gitlab-ci\.yml|^scripts/ci/.*|^\.github/.*|^\.config/nextest.toml
+      # excluding files from 'CI team' and 'FRAME coders' rules
+      exclude: ^\.gitlab-ci\.yml|^scripts/ci/.*|^\.github/.*|^\.config/nextest.toml|^frame/(?!.*(nfts/.*|uniques/.*|babe/.*|grandpa/.*|beef|merkle-mountain-range/.*|contracts/.*|election|nomination-pools/.*|staking/.*))
     min_approvals: 2
     teams:
       - core-devs
 
+  - name: FRAME coders
+    check_type: changed_files
+    condition:
+      include: ^frame/(?!.*(nfts/.*|uniques/.*|babe/.*|grandpa/.*|beef|merkle-mountain-range/.*|contracts/.*|election|nomination-pools/.*|staking/.*))
```

Comment:
This line includes everything under `frame` expect for those named sub directories, or partially named ones (`beef` will match `beefy` and `beefy-mmr`)
@kianenigma @ggwpez do you think there's more we want to exclude?